lib/mortar/list_command.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require_relative "command"
+require_relative "mixins/client_helper"
+require_relative "mixins/tty_helper"
+
+module Mortar
+  class ListCommand < Mortar::Command
+    include Mortar::ClientHelper
+    include Mortar::TTYHelper
+
+    option ['-q', '--quiet'], :flag, "only output shot names"
+
+    def execute
+      shots = {}
+
+      client.list_resources(labelSelector: LABEL).select{ |r|
+        r.metadata.labels&.dig(LABEL)
+      }.uniq{ |r|
+        # Kube api returns same object from many api versions...
+        "#{r.kind}/#{r.metadata.name}/#{r.metadata.namespace}"
+      }.each do |resource|
+        shot_name = resource.metadata.labels&.dig(LABEL)
+
+        if shots[shot_name]
+          shots[shot_name] += 1
+        else
+          shots[shot_name] = 1
+        end
+      end
+
+      if quiet?
+        shots.keys.each do |k|
+          puts k
+        end
+      else
+        table = TTY::Table.new %w(NAME RESOURCES), []
```

Comment:
```suggestion
  table = TTY::Table.new %w(NAME RESOURCES), shots.to_a
```

(can omit the loop below)